pkg/cmd/server/start.go

```diff
@@ -54,7 +56,11 @@ func (cfg Config) startMaster() error {
 		return err
 	}
 
-	glog.Infof("  Nodes: %v", cfg.NodeList)
+	glog.Infof("Nodes: %v", cfg.NodeList)
+
+	if strings.Contains(openshiftConfig.MasterAddr, "127.0.0.1") {
```

Comment:
If the BindAddr includes other IPs, then a container that knew where it was located would be able to communicate, wouldn't it?  We may not wire it up that way by default, but I'm pretty sure it could be done with a cleverly built .kubeconfig.